### PR TITLE
issue-22 newlines stripped form index, validateJS added to gulpfile

### PR DESCRIPTION
**Pull request by benmcosker**
newline error on src/index.js was preventing pipeline function combined with the fact that there was no validateJS processing happening in the `gulpfile.js`. Additionally there are some changes to pass the validation in an instantiation of the module object `AutoprefixPlugin` as validation requires cap case not camel case.

Verified function in the javascript tooling project pulling this commit into `package.json`:
````"pipeline-compile-less": "git://github.com/kenzanlabs/pipeline-compile-less#1fec55e2d80d58745fae9575ae89d8b3d5586411",````

cc: @obuckley-kenzan | @paul-barry-kenzan

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 var gulp = require('gulp');
 var compilePipeline = require('./src/index.js');
 var testPipeline = require('pipeline-test-node');
+var validatePipeline = require('pipeline-validate-js')();
```
Comment (obuckley-kenzan) [inline gulpfile.js]: pipeline validate JS was updated by @mrgardner  in PR #23 .  Please rebase and update accordingly.

File: gulpfile.js [hunk 2]
```diff
@@ -24,7 +25,7 @@ var config = {
   ],
 
   lessFiles: [
-   'test/fixtures/*.less'
+    'test/fixtures/*.less'
```
Comment (obuckley-kenzan) [inline gulpfile.js]: @benmcosker 
Just noticed, on line 32, I think it should be
```javascript
./*.js
```

Comment (benmcosker) [inline gulpfile.js]: my mistake, I had thought that this would be necessary to validate `.spec.js` files.

Comment (obuckley-kenzan) [inline gulpfile.js]: No problem, I would still double check by I think it should work.

Comment (benmcosker) [inline gulpfile.js]: good tip, catches `ESLintError`'s in testing on the `.spec.js` file.

Comment (benmcosker) [inline gulpfile.js]: This PR is necessary for pipeline to function. Ready for merge?

cc: @paul-barry-kenzan | @obuckley-kenzan

**Comment by paul-barry-kenzan** [comment]
This looks good to me.

**Comment by obuckley-kenzan** [comment]
resolves #22

**Pull request merged**